docs/io.cozy.files.md

```diff
@@ -213,7 +213,8 @@ They can have `metadata`:
 - `target.mime` is the mime-type of the destination of the link (when it is a
   file)
 - `target.app` is the slug of the destination app (internal links only).
-- `icon` {string?} contains the svg icon.   
+- `icon` {string?} contains the svg icon base64 encoded.
```

Comment:
The icon can be also a binary of an svg icon so maybe this description is more generic : `contains svg icon or base64 encoded image`